website/source/docs/virtualbox/networking.html.md

```diff
@@ -10,6 +10,22 @@ description: |-
 
 # Networking
 
+## VirtualBox Host-Only Networks
+
+By default, private networks are [host-only networks](https://www.virtualbox.org/manual/ch06.html#network_hostonly),
+because those are the easiest to work with. In VirtualBox, since you can
+create multiple host-only networks, it is also possible to specify which
+host-only network you want to the Vagrant Virtualbox provider to use for
+a given interface. To do this, use the `:name` symbol with the name of
+the host-only interface to use.
+
+```ruby
+Vagrant.configure("2") do |config|
+  config.vm.network "private_network", type: "dhcp",
+    :name => "vboxnet3"
```

Comment:
Small nitpick here, but this should be updated to `name: "vboxnet3"` instead of the hash style syntax, just so that we're consistent since `type` is defined this way in the example.

Reply:
Removed references to symbol syntax in 945e623f6d1166e82ac79e577410ada3a0fc3db9.